### PR TITLE
Add reload button to editor during development

### PR DESCRIPTION
**Pull request by fregante**
Screen Shot

### CODE REVIEW

File: src/devTools/editor/sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -176,6 +180,12 @@ const SidebarExpanded: React.FunctionComponent<
                   />
                 ))}
             </DropdownButton>
+
+            {process.env.ENVIRONMENT === "development" && (
```
Comment (twschiller) [inline src/devTools/editor/sidebar/Sidebar.tsx:184]: This is probably the right flag. We also have a real feature flag we could use for this if we also need it to show up in release/production builds

File: src/devTools/editor/sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -43,12 +43,17 @@ import styles from "./Sidebar.module.scss";
...
   faAngleDoubleLeft,
   faAngleDoubleRight,
+  faSync,
 } from "@fortawesome/free-solid-svg-icons";
 import { CSSTransition } from "react-transition-group";
 import cx from "classnames";
 import { CSSTransitionProps } from "react-transition-group/CSSTransition";
 import AuthContext from "@/auth/AuthContext";
 
+function reload() {
```
Comment (twschiller) [inline src/devTools/editor/sidebar/Sidebar.tsx:53]: There's no real benefit to defining this up here vs. just using an arrow function in the onClick handler. For buttons there's no performance penalty to an arrow function in the React props

**Review by twschiller — APPROVED** [review]

File: src/devTools/editor/sidebar/Sidebar.tsx (hunk 1 above)
Comment (fregante) [reply src/devTools/editor/sidebar/Sidebar.tsx:184]: >  We also have a real feature flag

What is that or where can I find an example?

Comment (twschiller) [reply src/devTools/editor/sidebar/Sidebar.tsx:180]: http://github.com/pixiebrix/pixiebrix-extension/blob/9df7d5bd8fb28464ee49a9169beda6580ff62f93/src/devTools/editor/tabs/editTab/dataPanel/DataPanel.tsx#L74-L74

We use the "page-editor-developer" feature flag:

```
const { flags } = useContext(AuthContext);
const showDeveloperTabs = flags.includes("page-editor-developer");
```

**Review by BALEHOK — APPROVED** [review]
Awesome! Especially with Chrome 97.

**Review by BLoe — APPROVED** [review]
Agree that it might be helpful to use the feature flag so the button will appear in production builds for us as well. Otherwise LGTM 👍

**Comment by fregante** [comment]
I added support via the pre-existing `page-editor-developer` flag, which feels like should cover this too. Merge if I used it correctly.

**Pull request merged**